src/types/venues.ts

```diff
@@ -245,6 +245,7 @@ export interface JazzbarVenue extends BaseVenue {
   host: {
     icon: string;
   };
+  enableJukebox?: boolean;
```

Comment:
😍  finally we start distinguishing properties between the venue types. mmmm. Thanks!